spec/mail/fields/unstructured_field_spec.rb

```diff
@@ -150,7 +151,7 @@
 
     it "should fold itself if it is non us-ascii" do
       @original = $KCODE if RUBY_VERSION < '1.9'
-      string = "This is あ really long string This is あ really long string This is あ really long string This is あ really long string This is あ really long string"
+      string = String.new("This is あ really long string This is あ really long string This is あ really long string This is あ really long string This is あ really long string")
       @field = Mail::UnstructuredField.new("Subject", string)
       if string.respond_to?(:force_encoding)
         string = string.force_encoding('UTF-8')
```

Comment:
I think we need to work to keep the literal string syntax where we can. We can do `string = string.dup.force_encoding(…)` in many of these cases.

However, this is another case where the `#force_encoding` look unneeded since the source encoding is already utf-8.